routes/routes.go

```diff
@@ -0,0 +1,55 @@
+package routes
+
+import (
+	"github.com/gofiber/fiber"
+	"learning_go/book"
+	"learning_go/database"
+)
+
+func GetBooks(c *fiber.Ctx) {
+	db := database.DB.Db
+	var books []book.Book
+	db.Find(&books)
+	c.JSON(books)
+}
+
+func GetBook(c *fiber.Ctx) {
+	id := c.Params("id")
+	db := database.DB.Db
+	var book book.Book
+	db.Find(&book, id)
+
+	if book.Name == "" {
+		c.Status(404).Send("No book found with the ID" + id)
```

Comment:
I'd probably send back a JSON payload of an error, this may send back a single string of the error message, which isn't valid JSON to consume on the client side. Unless they look for a 404 status first and THEN parse the body, this could cause issues on the client side.